source/basic.tex

```diff
@@ -5549,6 +5549,20 @@
 The standard and extended floating-point types
 are collectively termed \defnx{floating-point types}{type!floating-point}.
 \begin{note}
+A floating-point type can represent at least a subset of rational numbers.
```

Comment:
Maybe mention finite here. Apart from signed zero, the special cases listed below are not finite.

If you're trying to be exhaustive, there are also subnormals (but I don't think being exhaustive is necessary).

Reply:
I don't think a definition of finite belongs here, that's what 60559 is for. This should stay note-y.